stores/planship.ts

```diff
@@ -0,0 +1,258 @@
+import { defineStore, skipHydrate } from 'pinia'
+
+import { useUserStore } from '@/stores/user'
+import { useProjectsStore } from '@/stores/projects'
+
+import { Planship } from '@planship/fetch'
+
+async function getAccessToken() {
+  return fetch('/api/planshipToken').then((response) => response.text())
+}
+
+function getApiUrl() {
+  return typeof window === 'undefined' ? useRuntimeConfig().public.serverPlanshipApiUrl : useRuntimeConfig().public.clientPlanshipApiUrl
+}
+
+function createServerApiClient() {
+  // Create Planship client using API credentials (server side only)
+  return new Planship(
+    'clicker-demo',
+    getApiUrl(),
+    useRuntimeConfig().public.planshipApiClientId ,
+    useRuntimeConfig().planshipApiClientSecret,
+    useRuntimeConfig().public.webSocketUrl,
+  )
+}
+
+function createBrowserApiClient() {
+  return new Planship('clicker-demo', getApiUrl(), getAccessToken, useRuntimeConfig().public.webSocketUrl)
+}
+
+// This class should come autogenerated by Planship (as a snippet to copy/paste?)
```

Comment:
Agreed, but it doesn't today. This comment may be confusing without the context of our internal discussions. :)